motors_weg_cvw300.orogen

```diff
@@ -13,6 +13,9 @@ import_types_from "motors_weg_cvw300/FaultState.hpp"
 import_types_from "motors_weg_cvw300Types.hpp"
 
 using_task_library "iodrivers_base"
+using_library "usv_control"
+import_types_from "usv_control"
+import_types_from "usv_control/SaturationSignal.hpp"
```

Comment:
OK. No. We can't do that.

We'll have to have some `_base` package with the SaturationSignal type. We can't be adding dependency on usv_control on drivers like this.

Actually, a good candidate would be `control/orogen/control_base` ... maybe create `control/control_base` and `control/orogen/control_base` ?

The purpose of the _base package on the orogen side is to generate type handling for everyone (to avoid compiling it over and over again). See for instance the power_base package.

Reply:
I agree with this, once we will use `SaturationSignal` in other components too